Prepare for 1.4.2 (#1541)
microsoft/vscode-react-native: CHANGELOG.md

## 1.4.2
* Added support for debugging AppleTV apps, thanks to [Michele Bonazza(@micheleb)](https://github.com/micheleb) [#1498](https://github.com/microsoft/vscode-react-native/pull/1498)

> * Improved extension security [#1538](https://github.com/microsoft/vscode-react-native/pull/1538), [#1527](https://github.com/microsoft/vscode-react-native/pull/1527), [1547](https://github.com/microsoft/vscode-react-native/pull/1547)

Reviewer: ```suggestion
* Improved extension security [#1538](https://github.com/microsoft/vscode-react-native/pull/1538), [#1527](https://github.com/microsoft/vscode-react-native/pull/1527), [#1547](https://github.com/microsoft/vscode-react-native/pull/1547)
```

## 1.4.2

> * Added debugging support for AppleTV [#1498](https://github.com/microsoft/vscode-react-native/pull/1498)

Reviewer: ```suggestion
* Added support for debugging AppleTV apps [#1498](https://github.com/microsoft/vscode-react-native/pull/1498)
```

> * Added support for debugging AppleTV apps [#1498](https://github.com/microsoft/vscode-react-native/pull/1498)

Reviewer 2: ```suggestion
* Added support for debugging AppleTV apps, thanks to [Michele Bonazza(@micheleb)](https://github.com/micheleb) [#1498](https://github.com/microsoft/vscode-react-native/pull/1498)
```

* Improved extension security [#1538](https://github.com/microsoft/vscode-react-native/pull/1538)

> * Resolved confict with [vscode-jest](https://github.com/jest-community/vscode-jest) extension while fast refreshing React Native apps [#1530](https://github.com/microsoft/vscode-react-native/pull/1530)

Reviewer: ```suggestion
* Resolved a conflict with [vscode-jest](https://github.com/jest-community/vscode-jest) extension while fast refreshing React Native apps [#1530](https://github.com/microsoft/vscode-react-native/pull/1530)
```

* Migrated from `@expo/xdl` package to `xdl` package for debugging Expo applications [#1532](https://github.com/microsoft/vscode-react-native/pull/1532)
* Added multiroot project support [#1540](https://github.com/microsoft/vscode-react-native/pull/1540)
* Internal changes:

>     * Improve sending error events to Telemetry [#1526](https://github.com/microsoft/vscode-react-native/pull/1526)

Reviewer: ```suggestion
    * Improved sending error events to Telemetry [#1526](https://github.com/microsoft/vscode-react-native/pull/1526)
```
